process-automation/latest/model/interfaces.md

```diff
@@ -43,6 +43,16 @@ Custom end user actions can be created for the Digital Workspace. This enables c
 * A list of one or more [rules](#rules) to describe which nodes the action will be displayed on.
 * The [feature](#features) the action will be displayed on, for example the header or a context menu.
 
+### Auditing
```

Comment:
I would rather use an h2 header and move it to the bottom of the page, I don't think it should be inside the user interface modeling section

Reply:
done cheers